Makefile

```diff
@@ -7,7 +7,9 @@ INSTALL ?= install
 SASSC ?= sassc
 MKDIR ?= mkdir
 CONVERT ?= convert
-
+WEBPACK_DEV_SERVER := ./node_modules/webpack-dev-server/bin/webpack-dev-server.js
+WEBPACK_CONFIG := ./www/webpack.config.dev.js
+WEBPACK_PORT := 8080
 node_modules: package.json package-lock.json
```

Comment:
We should have a blank line between the end of the variables and the first target definition